payments/billing_sdk/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+  id(GradlePluginId.ANDROID_LIBRARY)
+  id(GradlePluginId.ANDROID_MODULE)
+  id(GradlePluginId.HILT)
+}
+
+android {
+  namespace = "com.aptoide.pt.billing_sdk"
```

Comment:
This doesn't look right, since this module is meant to be shared with the Wallet
Mb use `com.appcoins.billing.sdk`?